iOS/Source/Root/Extension/Extensions.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+import UIKit
+
+extension UIColor {
+    
+    open class var tokenWhite: UIColor {
```

Comment:
You can just rename it to `quickClockWhite` and the other color names with token too ✌️